src/arrayMethodJoin.js

```diff
@@ -5,7 +5,27 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
-    // write code here
+    let joinResult = '';
+
+    if (this.length === 0) {
```

Comment:
не потрібно перевіряти на 0
```suggestion
    if (!this.length) {
```